mlflow/keras.py

```diff
@@ -46,11 +47,15 @@ def _load_model(model_file):
 
 
 class _KerasModelWrapper:
-    def __init__(self, keras_model):
+    def __init__(self, keras_model, graph, sess=None):
         self.keras_model = keras_model
+        self._graph = graph
+        self._sess = sess
 
     def predict(self, dataframe):
-        predicted = pd.DataFrame(self.keras_model.predict(dataframe))
+        with self._graph.as_default():
+            with self._sess.as_default():
```

Comment:
If `self._sess` is `None`, which is the default value, this will fail.

Reply:
True. Right now this can not happen but I will remove the default value.